.gitignore

```diff
@@ -1,3 +1,7 @@
+# Packages
+package-lock.json*
+package.json*
```

Comment:
If your script requires any modules, then this shouldn't be in the gitignore. If you aren't using any node packages, you can delete these files.